migrations/env.py

```diff
@@ -0,0 +1,79 @@
+from logging.config import fileConfig
+
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+
+from pudl.metadata.classes import Package
+from pudl.workspace.setup import get_defaults
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+target_metadata = Package.from_resource_ids().to_sql()
+
+# other values from the config, defined by the needs of env.py,
+# can be acquired:
+# my_important_option = config.get_main_option("my_important_option")
+# ... etc.
+
+db_location = get_defaults()["pudl_db"]
```

Comment:
I think it would be helpful to log the database path here so folks know which db the migrations are being applied to.

Reply:
Also, I've been getting some unexpected behavior when working with `get_defaults()`.

```
PUDL_OUTPUT=/Users/bendnorman/catalyst/dagster-pudl-work/pudl_output
```
but `get_defaults()["pudl_db"]` returns:

```
/Users/bendnorman/catalyst/dagster-pudl-work/output/pudl.sqlite
```
Sounds like @e-belfer is having a similar issue.

Reply:
Ah, this is because of some funky stuff that @zschira and I left in from the old path derivation code. Easy fix, I think.